requirements/base.in

```diff
@@ -1 +1 @@
-Django ~= 3.0
```

Comment:
How is this change related to the PR?

Reply:
I remember about atomicity, but in this PR I added `coverage` and restructured requirements. It may be complicated to separate them.

Reply:
This file just doesn't need any changes at all. No need to separate anything from anywhere. You add other deps into a different lockfile.

Reply:
Marking this as resolved.